pkg/apis/servicecatalog/types.go

```diff
@@ -215,6 +212,9 @@ type InstanceStatus struct {
 	// an async operation started, it should be sent back to the broker
 	// on poll requests as a query param.
 	LastOperation string
+
+	// The URL of a web-based management user interface for the service instance
+	DashboardURL *string
```

Comment:
Anything optional should be a pointer in the API so that we can distinguish between set to empty and unset, documented here: https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#optional-vs-required